OZprivate/rawJS/OZTreeModule/src/tree_state.js

```diff
@@ -41,12 +41,12 @@ class TreeState {
   setup_canvas(canvas) {
     this._canvas = canvas;
     if (this._xp === null) {
-      this._xp = canvas.width/3;
-      this._yp = canvas.height - 200;
+      this._xp = canvas.clientWidth / 3;
+      this._yp = canvas.clientHeight - 200;
       this._ws = 1;   
     }
-    this.widthres = canvas.width;
-    this.heightres = canvas.height;
+    this.widthres = canvas.clientWidth;
```

Comment:
I was somewhat inclined to rename widthres to width everywhere, since it's not really a resolution anymore, but thought that might look a bit extreme

Reply:
Well, it's still *a* resolution, it just doesn't say what units that resolution is. I'd be tempted to rename them to ``.cssWidth`` or similar, but I'm not sure it's worth the fiddling & code churn.

I'm also slightly surprised that it's ``clientWidth`` we need here, but happy to take your word for it.